basic/13-decentralized-exchange/uniswap-v1-like/hardhat.config.js

```diff
@@ -1,7 +1,5 @@
-require("@nomiclabs/hardhat-waffle");
-const { utils } = require("ethers");
-const { GAS_PRICE } = require("./utils")
-
+require("@nomicfoundation/hardhat-toolbox");
+var ethers = require('ethers').ethers;
```

Comment:
I'm not sure it's better to write this way:

```suggestion
const { ethers } = require('ethers');
```

Reply:
yes,you are correct.thx